runner/execer/fake/intercept.go

```diff
@@ -0,0 +1,31 @@
+package fake
```

Comment:
The more I think about it, the more I like the general pattern to have:

bar/foo: the package that defines interfaces
bar/foo/foos: the package that contains simple implementations (fakes, mocks, composites, etc.)

This gets us two main benefits:
*) separation between interface and implementation
*) imports don't have lots of packages like "fake" or "memory" or "simple"